platform/consensus/ordering/pbft/performance_manager.h

```diff
@@ -35,6 +37,27 @@
 
 namespace resdb {
 
+ class PerformanceClientTimeout{
+ public:
+     std::string hash;
+     uint64_t timeout_time;
+
+     PerformanceClientTimeout(std::string hash_, uint64_t time_){
+         this->hash = hash_;
+         this->timeout_time = time_;
+     }
+
+     PerformanceClientTimeout(const PerformanceClientTimeout& other){
+         this->hash = other.hash;
+         this->timeout_time = other.timeout_time;
+     }
+
+     bool operator<(const PerformanceClientTimeout& other) const{
+         return timeout_time > other.timeout_time;
+     }
+ };
```

Comment:
```suggestion

public:
     PerformanceClientTimeout(std::string hash_, uint64_t time_);
     

     PerformanceClientTimeout(const PerformanceClientTimeout& other);

     bool operator<(const PerformanceClientTimeout& other) const;
     
      private:
     std::string hash_;
     uint64_t timeout_time_;
 };
```

Move the implementations to cpp.